README.md

```diff
@@ -1,3 +1,84 @@
-### module-1-hello-project
-This project is created to practice workflow in GitHub and use of Linters
+<a name="readme-top"></a>
 
+<div align="center">
+  <img src="cover-image.png" alt="logo" height="100" width="auto" />
+  <br/>
+
+  <h3><b>Module 1 Hello Microverse</b></h3>
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 About the project <a name="about-project"></a>
+
+This project is created to practice workflow in GitHub, pull Request and Linters
+
+**Hello Microverse** is a Hello Microverse web page to practice use of linters
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Use a style.css file to beautify the page.
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> there's no live demo yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You can follow the Microverse repo's about linter to learn about this topic
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jorge Camargo**
+
+- GitHub: [@jicamargo](https://github.com/jicamargo)
+- Twitter: [@ji_camargo](https://twitter.com/ji_camargo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jorgecamargog)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project! 
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse Team
+- All my fellow micronauts
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Concerning the license file and the linkage to it: I believe that what the previous reviewers were trying to say in simple terms is that you do not have a license file in your project set up, hence when this link is pressed, it redirects to a 404 page not found instead of your license. To fix this, I suggest creating a file name LICENSE (All in uppercase) at the root directory of your project and then copy the content from [here](https://github.com/microverseinc/readme-template/blob/master/MIT.md) into that file.

Note: add your name in the place provided for you in the license.

Consider implementing this correctly to make the readme more professional.

Reply:
Think you @DuaneDave . I added it now.

Reply:
### Thank you @DuaneDave
#### please help me:
 I have checked all previous reviews. 
-It's missing someone? where do I find it? 
I have used all my review request. :-O 
 Thank you